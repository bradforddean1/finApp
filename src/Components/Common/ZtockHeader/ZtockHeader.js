import React from "react";
import "./ZtockHeader.scss";
import PropTypes from "prop-types";

/**
 * Logo'd header component
 * @component
 */
function ZtockHeader(props) {
	return <div className="ZtockHeader">{/* content goes here */}</div>;
}

export default ZtockHeader;
