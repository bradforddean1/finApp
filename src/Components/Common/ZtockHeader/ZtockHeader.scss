.ZtockHeader {
	height: 100%;
	display: flex;
	flex-direction: column;
	justify-content: center;
	background-repeat: no-repeat;
	background-size: 100%;
}

.ZtockHeader h1 {
	text-align: center;
	margin-bottom: 20px;
}

.ZtockHeader p {
	font-family: "hind";
	font-weight: 500;
	font-size: 24px;
	line-height: 32px;
	color: #46d16d;
}
