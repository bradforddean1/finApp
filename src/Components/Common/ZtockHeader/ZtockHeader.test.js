import React from "react";
import ReactDOM from "react-dom";
import ZtockHeader from "./ZtockHeader";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

describe("ZtockHeader Component", () => {
	it("Renders", () => {
		const div = document.createElement("div");
		ReactDOM.render(<ZtockHeader />, div);
		ReactDOM.unmountComponentAtNode(div);
	});

	it("Renders the Default State", () => {
		const wrapper = shallow(<ZtockHeader />);
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
