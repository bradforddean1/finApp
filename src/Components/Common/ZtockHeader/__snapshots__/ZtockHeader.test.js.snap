// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ZtockHeader Component Renders the Default State 1`] = `
<div
  className="ZtockHeader"
/>
`;
